charmhelpers/fetch/ubuntu.py

```diff
@@ -217,13 +219,25 @@ def filter_missing_packages(packages):
 
 
 def apt_cache(in_memory=True, progress=None):
-    """Build and return an apt cache."""
-    from apt import apt_pkg
-    apt_pkg.init()
-    if in_memory:
-        apt_pkg.config.set("Dir::Cache::pkgcache", "")
-        apt_pkg.config.set("Dir::Cache::srcpkgcache", "")
-    return apt_pkg.Cache(progress)
+    """Shim returning an object simulating the apt_pkg Cache.
+
+    :param in_memory: Kept for compability reasons, has no effect.
```

Comment:
You could just use `*args` in the function signature to show that there might be args, but that you are ignoring them.  (or even `*_` which would make it even more obvious)?

Reply:
:+1: